pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -357,6 +366,33 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 		return err
 	}
 
+	// Set sysctls if requested, using the CNI tuning plugin
```

Comment:
Looks like we have an issue with this code which was meant for the rkt case. But kubenet is used also for Docker on GCE, and fails to set non-`net.*` sysctls.